server/server.go

```diff
@@ -236,6 +235,12 @@ func (s *Server) URLs() []string {
 	return s.cfg.endpoints
 }
 
+// URL returns the resolved opc endpoint that the server is listening on.
+// Useful when a :0 port was provided.
```

Comment:
Let's remove the "Useful..." comment and update the `server.EndPoint` option comment instead, e.g.

```
Use `0` to choose a random free port.
```

Reply:
done in https://github.com/gopcua/opcua/pull/798/commits/eafb6cfb105fd1cb8981b3c353b43f193fc8b520